tensorflow/lite/kernels/slice_test.cc

```diff
@@ -119,6 +119,21 @@ TEST_P(SliceOpTest, InputFloat) {
   EXPECT_THAT(m.GetOutput(), ElementsAreArray({2, 3, 4}));
 }
 
+TEST(SliceOpTest, In4D) {
+  SliceOpModel<float, int32_t> m({2, 3, 2, 2}, {4}, {4}, TensorType_INT32,
```

Comment:
We already support 4D? Shouldn't there be something like a 5D test?

Reply:
Please refer: https://github.com/tensorflow/tensorflow/pull/27113#discussion_r271167137